modal.js

```diff
@@ -1,14 +1,14 @@
 var modal = document.querySelector("#modal");
```

Comment:
hmm sorry I missed this before, all these vars are in global scope! Can we wrap them up in a function?